src/services/database/serverless.yml

```diff
@@ -0,0 +1,82 @@
+service: ${self:custom.project}-database
+
+frameworkVersion: "3"
+
+package:
+  individually: true
+
+plugins:
+  - serverless-stack-termination-protection
+  - "@stratiformdigital/serverless-s3-security-helper"
+  - "@stratiformdigital/serverless-iam-helper"
+  - serverless-bundle
+
+provider:
+  name: aws
+  region: us-east-1
+  iam:
+    role:
+      path: /delegatedadmin/developer/
+      permissionsBoundary: arn:aws:iam::${aws:accountId}:policy/cms-cloud-admin/developer-boundary-policy
+      statements:
+        - Effect: Allow
+          Action:
+            - logs:CreateLogGroup
+            - logs:CreateLogStream
+            - logs:PutLogEvents
+            - dynamodb:PutItem
+            - dynamodb:UpdateItem
+            - dynamodb:DeleteItem
+            - dynamodb:BatchWrite*
```

Comment:
how much of this do we actually need?

Reply:
We actually need all.

Reply:
The log permissions have been removed. The arn has also been updated to a specific table.


        - Effect: Allow
          Action:
            - dynamodb:Query
            - dynamodb:GetItem
            - dynamodb:PutItem
            - dynamodb:UpdateItem
            - dynamodb:DeleteItem
            - dynamodb:Get*
            - dynamodb:BatchWrite*
            - dynamodb:BatchGet*
          Resource: "arn:aws:dynamodb:*:*:table/${self:custom.tableName}"